model/host/host.go

```diff
@@ -1420,34 +1420,14 @@ func (h *Host) clearRunningTaskWithFunc(doUpdate func(update bson.M) error) erro
 	return nil
 }
 
-// UpdateRunningTask updates the running task in the host document and logs an
```

Comment:
This is gonna cause a conflict with #6236 which already fixed this + refactored `UpdateRunningTaskWithContext`. I would just leave this change out of the PR.

Reply:
oh sweet i'll just add the logging changes then